cwltool/job.py

```diff
@@ -323,17 +353,18 @@ def get_from_requirements(self, r, req, pull_image, dry_run=False):
         # type: (Dict[Text, Text], bool, bool, bool) -> Text
         pass
 
+
+        # type: (bool, bool, bool, Text, **Any) -> None
```

Comment:
I think the line above needs to be removed, or put where it belongs.

Reply:
Done